docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  web:
+    image: nginx:1.19-alpine
+    container_name: intern-web
+    working_dir: /application
+    restart: unless-stopped
+    tty: true
+    volumes:
+      - .:/application
+      - ./environment/nginx.conf:/etc/nginx/conf.d/default.conf
+    ports:
+      - "${EXTERNAL_WEBSERVER_PORT}:80"
+
+services:
+  php:
+    image: ghcr.io/blumilksoftware/php:${PHP_VERSION}
```

Comment:
I think that PHP version shouldn't be dependant on developers environment. You should provide minimal semver here.